dotnet/Face/sdk/detect.cs

```diff
@@ -1,78 +1,112 @@
-// <basic1>
-IList<DetectedFace> faces = await faceClient.Face.DetectWithUrlAsync(imageUrl, true, false, null);
-// </basic1>
+using System;
+using System.Collections.Generic;
+using System.Linq;
+/* Add a reference to the WindowsBase .NET Framework assembly.
+* Note: Currently, that means this project must be created to
+* target the .NET Framework and not .NET Core.
+*/
+using System.Windows;
+// Install NuGet package Microsoft.Azure.CognitiveServices.Vision.Face.
+using Microsoft.Azure.CognitiveServices.Vision.Face;
+using Microsoft.Azure.CognitiveServices.Vision.Face.Models;
 
-// <basic2>
-foreach (var face in faces)
+namespace ConsoleApp1
 {
-    string id = face.FaceId.ToString();
-    FaceRectangle rect = face.FaceRectangle;
-}
-// </basic2>
+    class Program
+    {
+        static string SUBSCRIPTION_KEY = Environment.GetEnvironmentVariable("FACE_SUBSCRIPTION_KEY");
+        static string ENDPOINT = Environment.GetEnvironmentVariable("FACE_ENDPOINT");
 
-// <landmarks1>
-IList<DetectedFace> faces = await faceClient.Face.DetectWithUrlAsync(imageUrl, true, true, null);
-// </landmarks1>
+        async static void Quickstart()
+        {
+            IFaceClient faceClient = new FaceClient(new ApiKeyServiceClientCredentials(SUBSCRIPTION_KEY)) { Endpoint = ENDPOINT };
 
-// <landmarks2>
-foreach (var face in faces)
-{
-    var landmarks = face.FaceLandmarks;
+            var imageUrl = "https://raw.githubusercontent.com/Azure-Samples/cognitive-services-sample-data-files/master/ComputerVision/Images/faces.jpg";
 
-    double noseX = landmarks.NoseTip.X;
-    double noseY = landmarks.NoseTip.Y;
+            // <basic1>
+            IList<DetectedFace> faces = await faceClient.Face.DetectWithUrlAsync(url: imageUrl, returnFaceId: true, detectionModel: DetectionModel.Detection02);
```

Comment:
have you tested this code? Detection02 don't support faceLandmarks, so the following code will fail/access null object etc.

Reply:
I apologize. I thought I had tested it, but it seems not. The first call to DetectWithUrlAsync is only used to get FaceRectangle so it can use detection model 2. The second and third calls, which are used to get landmarks and attributes, I changed to detection model 1.